sdks/python/apache_beam/transforms/sql_test.py

```diff
@@ -66,20 +66,8 @@ class SqlTransformTest(unittest.TestCase):
         --tests apache_beam.transforms.sql_test \\
         --test-pipeline-options="--runner=FlinkRunner"
   """
-  @staticmethod
-  def make_test_pipeline():
-    path_to_jar = subprocess_server.JavaJarServer.path_to_beam_jar(
-        ":sdks:java:extensions:sql:expansion-service:shadowJar")
-    test_pipeline = TestPipeline()
-    # TODO(BEAM-9238): Remove this when it's no longer needed for artifact
-    # staging.
-    test_pipeline.get_pipeline_options().view_as(DebugOptions).experiments = [
-        'jar_packages=' + path_to_jar
```

Comment:
Can we update 'validate_runner_xlang_test.py' for Spark and Flink as well or do we need additional changes before that.
jar_packages cannot be completely removed yet since Dataflow needs it.

Reply:
Done.